include/Dialect/Btor/IR/BtorOps.td

```diff
@@ -193,8 +194,8 @@ class BtorArithmeticOp<string mnemonic, list<Trait> traits = []> :
 //     <op> %0, %1 : i32
 class BtorBinaryOp<string mnemonic, list<Trait> traits = []> :
     BtorArithmeticOp<mnemonic, !listconcat(traits, [SameOperandsAndResultType])>,
-    Arguments<(ins SignlessIntegerLike:$lhs, SignlessIntegerLike:$rhs)>,
-    Results<(outs SignlessIntegerLike:$result)> {
```

Comment:
This is good..make sure that all the operations are changed to work exclusively with our custom types